pvs/default/savedsearches.conf

```diff
@@ -3,67 +3,67 @@ dispatch.earliest_time = -24h
 dispatch.latest_time = now
 displayview = flashtimeline
 request.ui_dispatch_view = flashtimeline
-search = eventtype="pvs" | top src
+search = eventtype=nnm | top src
 
 [Top Destination IP]
 dispatch.earliest_time = -24h
 dispatch.latest_time = now
 displayview = flashtimeline
 request.ui_dispatch_view = flashtimeline
-search = eventtype="pvs" | top dest
+search = eventtype=nnm | top dest
 
-[PVS Top Destination by Country]
+[NNM Top Destination by Country]
 dispatch.earliest_time = -24h@h
 dispatch.latest_time = now
 displayview = charting
 request.ui_dispatch_view = charting
-search = eventtype="pvs" | iplocation dest | top Country limit=10
+search = eventtype=nnm | iplocation dest | top Country limit=10
 vsid = g9yn5fso
 
-[PVS Top Source by Country]
+[NNM Top Source by Country]
 dispatch.earliest_time = -24h@h
 dispatch.latest_time = now
 displayview = charting
 request.ui_dispatch_view = charting
-search = eventtype="pvs" | iplocation src | top Country limit=10
+search = eventtype=nnm | iplocation src | top Country limit=10
 
-[PVS Top 10 Events]
+[NNM Top 10 Events]
 auto_summarize.dispatch.earliest_time = -1d@h
 description = Top events last 24hrs
 dispatch.earliest_time = -24h
 dispatch.latest_time = now
 displayview = flashtimeline
-search = eventtype="pvs" | top limit=10 PVS_eventname
+search = eventtype=nnm | top limit=10 plugin_name
```

Comment:
did PVS_eventname become plugin_name?

Reply:
Yes,  I switched the field naming conventions to match what they are called out in the pvs syslog documentation.  https://docs.tenable.com/nnm/5_4/Content/StandardSyslogMessageTypes.htm